### PR TITLE
update_middleContainer

### PR DESCRIPTION
**Pull request by alex-ns-nl**
To update the middleContainer and avoid abusing the float element, I added the following to "About me" and "My Skills":
- added image "geometricalShape"
- re-used the .positionAbsolute class

### CODE REVIEW

**Comment by anouq** [comment]
why did you rename `index.html` to `update_middleContainer`?

and even if you wanted to rename it, you forgot the file extension (`.html`). so now it's not even recognized as an html file :)

File: CSS/styles.css [hunk 1]
```diff
@@ -66,12 +69,17 @@ p {
 }
 
 .aboutMe {
-  float: left;
-  display: inline-block;
+  left: 0;
```
Comment (anouq) [inline CSS/styles.css]: `left: 0;` doesn't seem to be doing anything, so this property can be removed.

**Review by anouq — CHANGES_REQUESTED** [review]

File: CSS/styles.css (hunk 1 above)
Comment (alex-ns-nl) [reply CSS/styles.css]: You're right. I've removed the `aboutMe` class

**Review by anouq — APPROVED** [review]
nice job! :)

**Pull request merged**